templates/gotify/0/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '2'
+services:
+  gotify:
+    image: cthit/gotify
+    environment:
+      GOTIFY_PORT: '8080'
+      GOTIFY_PRE-SHARED-KEY: SecretShareKey
+      GOTIFY_DEBUG-MODE: 'false'
+      GOTIFY_GOOGLE-MAIL.KEYFILE: gapps.json
+      GOTIFY_GOOGLE-MAIL.ADMIN-MAIL: admin@chalmers.it
```

Comment:
I don't know how tedious this is, but adding these as [configuration options](https://screenshotscdn.firefoxusercontent.com/images/eeab0b3a-e821-4d5f-850c-c0da3e505d0c.png) in rancher could simplify deployment.